content/spin/rust-components.md

```diff
@@ -106,12 +106,12 @@ id = "hello"
 source = "target/wasm32-wasi/release/spinhelloworld.wasm"
 allowed_http_hosts = [ "some-random-api.ml" ]
 [component.trigger]
-route = "/hello"
+route = "/wilcard"
```

Comment:
Thanks @robotastic for aligning these docs more closely with the actual Rust code that is offered. As @itowlson suggests, I also agree that we could/should call this outbound (instead of wildcard). Thanks for updating this from the generic `hello`. You are correct in saying that there needs to be a change/update. Thanks for your input and effort.
```suggestion
route = "/outbound"
```